src/lib/util.ts

```diff
@@ -1,17 +1,19 @@
 const getNestedObject = (parent: any, path?: string): any => {
```

Comment:
Please replace this with lodash (or something similar). I am happy if you want to wrap the lodash methods in the util so that we can switch the implementations as required, but doing so will reduce the chance of bugs/errors being introduced.

Reply:
since its the only place we used it, I removed the util function